statistics/dump.go

```diff
@@ -71,8 +72,10 @@ func (h *Handle) DumpStatsToJSON(dbName string, tableInfo *model.TableInfo) (*JS
 		Count:        tbl.Count,
 		ModifyCount:  tbl.ModifyCount,
 	}
+
 	for _, col := range tbl.Columns {
-		hist, err := col.ConvertTo(new(stmtctx.StatementContext), types.NewFieldType(mysql.TypeBlob))
+		sc := &stmtctx.StatementContext{TimeZone: time.UTC}
```

Comment:
I'm not sure that use UTC is suitable for dump?

Reply:
@lamxTyler PTAL

Reply:
@lysu what's the default time zone before this PR?